charts/rancher-gatekeeper-operator/v3.1.0-beta.700/Chart.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: v1
+description: A Helm chart for Gatekeeper
+name: gatekeeper-operator
```

Comment:
Since all the other charts are named `rancher-X`, should we update this chart to be consistent?